multi-node/aws/pkg/config/config_test.go

```diff
@@ -309,3 +314,165 @@ releaseChannel: non-existant #this release channel will never exist
 	}
 
 }
+
+func TestNodeDrainerConfig(t *testing.T) {
+
+	validConfigs := []struct {
+		conf        string
+		nodeDrainer NodeDrainer
+	}{
+		{
+			conf: `
+`,
+			nodeDrainer: NodeDrainer{
+				Enabled:        false,
+				KubectlImage:   "mumoshu/kubectl:v1.2.2",
+				KubectlCommand: "kubectl",
```

Comment:
When the default Kubectlmage and KubectlCommand values are changed when the `/hyperkube kubectl` issue is fixed, we'll have to edit tests like to reflect the changes to defaults. 

I would like to see constants `defaultHyperkubeImage` and `defaultKubectlCommand`:
* defined in `config.go`
* applied as defaults to the Cluster struct [here](https://github.com/coreos/coreos-kubernetes/blob/master/multi-node/aws/pkg/config/config.go#L30)
* used in the config tests like this which infer the default value